teaching_platform/users/db_handler.py

```diff
@@ -46,6 +49,13 @@ def get_user(user_id):
 
     return user
 
+def get_user_by_email(email):
+    user = User.query.filter_by(email=email).first()
+
```

Comment:
I think we can remove empty lines in this function

Reply:
Done